### PR TITLE
Add whitepaper to Overview page 

### PR DESCRIPTION
**Pull request by helenlly**
## What changes were proposed in this pull request?

1) re-org Overview page: 
  Overview -> Overview :
                                    -- ReadMe 
                                    -- White Paper
                                        
2) add whitepaper.md
3) add wp pics into folder /Img/WP

## How was this patch tested?

test in local

### CODE REVIEW

File: docs/docs/whitepaper.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+<sup>ǂ</sup> Work was done when the author worked at Intel
+
+
+---
+
+## **Abstract**
+
+In this paper, we present BigDL, a distributed deep learning framework for Big Data platforms and workflows. It is implemented on top of Apache Spark, and allows users to write their deep learning applications as standard Spark programs (running directly on large-scale big data clusters in a distributed fashion). It provides an expressive, “data-analytics integrated” deep learning programming model, so that users can easily build the end-to-end analytics + AI pipelines under a unified programing paradigm; by implementing an *AllReduce* like operation using existing primitives in Spark (e.g., shuffle, broadcast, and in-memory data persistence), it also provides a highly efficiently “parameter server” style architecture, so as to achieve highly scalable, data-parallel distributed training. Since its initial open source release, BigDL users have built many analytics and deep learning applications (e.g., object detection, sequence-to-sequence generation, neural recommendations, fraud detection, etc.) on Spark.
```
Comment (hkvision) [inline docs/docs/whitepaper.md]: highly efficient?

**Comment by helenlly** [comment]
correct a space error

File: docs/docs/whitepaper.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+
+* **Next, another “parameter synchronization” job is launched;** each task n in the “parameter synchronization” job is responsible for managing the nth partition of the parameters, just like a parameter server (as shown in Figure 6). Specifically, the nth partition of the gradients (from all the tasks of the previous “model forward-backward” job) are first shuffled to task n, which then aggregates (sums) these gradients, and applies the updates to the nth partition of the weights (using the specific optimization method), as illustrated in Figure 5.
+
+```
+For each task n in the ”parameter synchronization” job
+   shuffle the nth partition of all gradients to this task
+   aggregate (sum) the gradients
+   updates the nth partition of the weights
+   broadcast the nth partition of the updated weights
+
```
Comment (hkvision) [inline docs/docs/whitepaper.md]: remove this line?

File: docs/docs/whitepaper.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+On the other hand, efficient and distributed training of deep neural networks would necessitate very different operations (such as fine-grained data access and in-place data mutation [3]). In this section, we describe in details how BigDL supports highly efficient and scalable distributed training, directly on top of the data parallel and functional compute model of Spark (in addition to various optimizations for model inference).
+
+### **3.1. Data-parallel training**
+
+To train a deep neural network model across the cluster, BigDL provides data-parallel training on Spark using synchronous mini-batch SGD, which is shown to achieve better scalability and efficiency (in terms of time-to-quality) compared to asynchronous training [29][30]. The distributed training in BigDL is implemented as an iterative process, as illustrated in Figure 3; each iteration runs a couple of Spark jobs to first compute the gradients using the current mini-batch, and then make a single update to the parameters of the neural network model.
+
+```
+for (i <- 1 to N) {
+  //”model forward-backward” job
```
Comment (hkvision) [inline docs/docs/whitepaper.md]: " vs ”?
